cli/kubectl-hyperkube

```diff
@@ -22,9 +55,21 @@ def cli(ctx, config):
     ctx.ensure_object(dict)
     ctx.obj['hyperkube_config'] = _load_config(config)
     ctx.obj['url'] = ctx.obj['hyperkube_config']['url']
+
+    u = urlparse(ctx.obj['url'])
+    ctx.obj['hostname'] = u.netloc
+
     ctx.obj['stage'] = ctx.obj['hyperkube_config']['stage']
-    headers = {'X-Api-Key': ctx.obj['hyperkube_config']['x_api_key']}
+    ctx.obj['region'] = ctx.obj['hyperkube_config'].get('region', "us-west-2")
+    ctx.obj['aws_profile'] = ctx.obj['hyperkube_config'].get(
+        'aws_profile', "default")
+    headers = {'X-Api-Key': ctx.obj['hyperkube_config'].get('x_api_key', '')}
```

Comment:
A blank api key produces an error on the request? Should the code check the api key before executing?

Reply:
It doesn't produce an error per my testing, but we could avoid setting it if not given. Let me make that change.